### ISSUE TITLE
Support for {% include "foo.stencil" %}

### ISSUE THREAD

**Issue by IanKeen** [SwiftGen/SwiftGen#950]
Right now if I am using a template:
```stencil
func doSomething() {
    sharedCode()
    something()
}
```

and 

```stencil
func doSomethingElse() {
    sharedCode()
    somethingElse()
}
```

right now I have to include `sharedCode` either in each template or as part of the project. Neither of these are ideal, the former adds a maintenance burden the latter makes it harder to setup for distribution.

What would be nice is to be able to keep `func sharedCode() { ... }` in its own file and include it where ever I need it like:

```stencil
{% include "Shared.stencil" %}

func doSomething() {
    sharedCode()
    something()
}
```

When attempting this now it fails with the following:
```
swiftgen: error: Template named `Shared.stencil` does not exist. No loaders found
```

**Comment by djbe** [comment]
@IanKeen I've implemented the necessary changes in #959, see if you can give that a try?

**Comment by djbe** [comment]
Implemented by SwiftGen 6.6.0, which has been released.
